broadinstitute/pearl
# JN-302 survey review mode

To make Whitney and Jeff's lives easier, this adds a review mode button that is visible in sandbox and irb environments.
![image](https://user-images.githubusercontent.com/2800795/233710420-28d09f27-43a3-44fa-9109-813ff810535d.png)

Switching it on shows all questions regardless of visibility rules, and allows paging ahead without validation.  Validation rules will still be required if the user attempts to submit the form.

TO TEST:
1. go to participant app, sign in as consented@test.com
2. go to the basics survey. 
3. turn on review mode
4. observe that now you don't have to fill out name to advance
